Sources/PackageCollectionSigner/PackageCollectionSign.swift

```diff
@@ -80,7 +80,9 @@ public struct PackageCollectionSign: ParsableCommand {
             try localFileSystem.copy(from: rootCertPath, to: tmpDir.appending(component: rootCertFilename))
 
             // Sign the collection
-            let signer = signer ?? PackageCollectionSigning(trustedRootCertsDir: tmpDir.asURL, callbackQueue: DispatchQueue.global())
+            let signer = signer ?? PackageCollectionSigning(trustedRootCertsDir: tmpDir.asURL,
+                                                            observabilityScope: ObservabilitySystem { _, _ in }.topScope,
```

Comment:
@tomerd Not sure an `ObservabilitySystem` initialized this way would do anything useful (does it log at the minimum?), but it doesn't seem like there's a basic `ObservabilitySystem` available?

Reply:
https://github.com/apple/swift-package-collection-generator/pull/40/commits/ef57499608b200075f14f4bf04bdbaa48312aa88